src/MarketingPageAcceptanceTests.TestData/ContactDetails/IContactDetail.cs

```diff
@@ -1,11 +1,21 @@
-﻿namespace MarketingPageAcceptanceTests.TestData.ContactDetails
+﻿using System.Collections.Generic;
+
+namespace MarketingPageAcceptanceTests.TestData.ContactDetails
 {
     public interface IContactDetail
```

Comment:
do we really need this interface? In my branch when i was making changes we could just use the concrete class direct

Reply:
There are 2 different contact detail tables in the data model, so this would enable us to use a whole bunch of methods freely between the 2 without extra re-writes